### PR TITLE
adding spin-orbit coupling to process label

### PR DESCRIPTION
**Pull request by AndresOrtegaGuerrero**
image

### CODE REVIEW

**Comment by AndresOrtegaGuerrero** [comment]
image

File: src/aiidalab_qe/app/submission/model.py [hunk 1]
```diff
@@ -98,6 +98,16 @@ def update_process_label(self):
...
         )
+
+        soc_parameters = self.input_parameters["advanced"]["pw"]["parameters"][
+            "SYSTEM"
+        ].get("lspinorb", False)
+
+        if soc_parameters:
+            soc_info = ", spin-orbit coupling"
+        else:
+            soc_info = ""
```
Comment (edan-bainglass) [inline src/aiidalab_qe/app/submission/model.py:109]:
```suggestion
        soc_info = ", spin-orbit coupling" if soc_parameters else ""
```
A bit more concise. If using VS Code, consider installing the Sourcery extension. It'll recommend (and optionally apply) best practices as you code.

File: src/aiidalab_qe/app/submission/model.py [hunk 2]
```diff
@@ -109,7 +119,7 @@ def update_process_label(self):
         if workchain_data["spin_type"] != "none":
             protocol_and_magnetic_info += ", magnetic"
         properties_info = f"→ {', '.join(properties)}" if properties else ""
-        label = f"{structure_label} [{relax_info}, {protocol_and_magnetic_info}] {properties_info}".strip()
+        label = f"{structure_label} [{relax_info}, {protocol_and_magnetic_info}{soc_info}] {properties_info}".strip()
```
Comment (edan-bainglass) [inline src/aiidalab_qe/app/submission/model.py:122]: Consider doing
```python
label_parts = [
    relax_info,
    protocol_and_magnetic_info,
    soc_info,
    ...,  # future info bits :)
]
```
then here
```python
label = f"{structure_label} [{', '.join(label_parts)}] {properties_info}.strip()
```
Makes for easy future expansion 🙂

**Review by edan-bainglass — APPROVED** [review]
LGTM!

**Pull request merged**